src/docfx/Errors.cs

```diff
@@ -483,9 +483,6 @@ public static Error MonikerRangeUndefined(SourceInfo? source)
             /// or moniker-zone defined in article.md has no intersection with file-level monikers.
             /// </summary>
             /// Behavior: ✔️ Message: ❌
-            public static Error MonikeRangeOutOfScope(SourceInfo<string?> source, IEnumerable<string> zoneLevelMonikers, IEnumerable<string> fileLevelMonikers)
```

Comment:
Per offline discussion, the out of scope moniker range can be warning for moniker zone, but it should has its own error code, not sharing with the file level one anymore.

cc @herohua for awareness